chart/values_nightly.yaml

```diff
@@ -0,0 +1,126 @@
+defaultAirflowRepository: 242864343139.dkr.ecr.ap-northeast-1.amazonaws.com/sendbird/airflow
+defaultAirflowTag: v0.0.1-dev
+
+ingress:
+  web:
+    annotations:
+      alb.ingress.kubernetes.io/certificate-arn: arn:aws:acm:ap-northeast-1:242864343139:certificate/d6930331-0470-4d85-90c4-7b095914bf1f
+    host: airflow-nightly.dp-dev.sendbirdtest.com
+
+config:
+  google:
+    client_id: '1091187390014-u2l759rm5nu28ujik0glt6l214g5abm9.apps.googleusercontent.com'
+    client_secret: 'Bkxy_IQOVsXBAbTIuJkVzpLM'
+    domain: sendbirdtest.com
+
+webserver:
+  webserverConfig: |
+    # -*- coding: utf-8 -*-
```

Comment:
This configuration python file is necessary if we use rbac of Airflow. We could extract this as a separate file and include it in our docker image further, under AIRFLOW_HOME path.